core/templates/components/forms/schema-based-editors/schema-based-editor.component.ts

```diff
@@ -100,11 +98,7 @@ export class SchemaBasedEditorComponent
   ngAfterViewInit(): void {
     let angularJsFormController: angular.IFormController | undefined = (
       angular?.element(this.elementRef.nativeElement).controller('form'));
-    // This throws "Object is possibly undefined." The type undefined
-    // comes here from NgForm dependency. We need to suppress this
-    // error because of strict type checking.
-    // @ts-ignore
-    this.form.statusChanges.subscribe((validationStatus) => {
+    this.form.statusChanges?.subscribe((validationStatus) => {
```

Comment:
Ditto, what are the implications of the '?'.

Reply:
Changed it now with an explanation.